package.json

```diff
@@ -89,6 +99,7 @@
         "@rollup/plugin-babel": "^6.0.0",
         "@rollup/plugin-commonjs": "^24.0.0",
         "@rollup/plugin-node-resolve": "^15.0.0",
+        "@rollup/plugin-terser": "^0.2.1",
```

Comment:
nice work on moving off the deprecated package, although there's a newer version as of this morning.